restart_services.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+FILE=".env"
+inotifywait -m -e close_write $FILE |
+while read EVENT;
+do 
+  echo $EVENT      
+  echo ".env file changes detected. Restarting pipeline services..."
+  docker-compose down
+  docker-compose up --build
```

Comment:
Might be worth it to add `--no-cache` as well?

Reply:
Yes, added the flag.